docs/content/latest/explore/ysql-language-features/tablespaces.md

```diff
@@ -13,154 +13,220 @@ isTocNested: true
 showAsideToc: true
 ---
 
-This document provides an overview of tablespaces in YSQL and demonstrates how they can be used to specify data placement in the cloud. 
+This document provides an overview of YSQL Tablespaces and demonstrates how they can be used to specify data placement for tables and indexes in the cloud.
 
-This document guides you through the process of creating the following:
+## Overview
 
-- Multi-zone and single-zone tablespaces.
-- Tables that use multi-zone and single-zone tablespaces.
-- Indexes for tables that use multi-zone and single-zone tablespaces.
+In a distributed cloud-native database like YugabyteDB, the location of tables/indexes plays a very important role in achieving optimal performance for any workload. The below diagram illustrates the ping latencies amongst nodes in a geo-distributed cluster. It is very apparent that nodes closer to each other can communicate with visibly lesser latency than nodes physically far away from each other.
```

Comment:
tables/indexes >> tables and indexes   (slashes between words prevent accessibility readers from correctly interpreting text)